### ISSUE TITLE
Wish: Apple Fitness integration

### ISSUE THREAD

**Issue by voltagex** [merlos/iOS-Open-GPX-Tracker#255]
This is probably a long shot.

https://developer.apple.com/documentation/healthkit/hkworkoutroute

It'd be nice to have runs tracked in GPX Tracker count towards my move goal.

**Comment by jp-u** [comment]
Agreed. Would be great if you could select the [workout type] (https://support.apple.com/en-us/105089) for a given tracking.
